projects/admin/src/app/frontpage/frontpage.component.ts

```diff
@@ -27,27 +27,26 @@ import { LocalStorageService } from '@rero/ng-core';
 })
 export class FrontpageComponent implements OnInit {
   /** Lists of entries */
-  userServices = { title: {}, entries: [] };
-  catalog = { title: {}, entries: [] };
-  acquisitions = { title: {}, entries: [] };
-  adminMonitoring = { title: {}, entries: [] };
+  linksMenu: any;
 
   /** Constructor
-   * @param _translateService - TranslateService
    * @param _userService - User Service
    * @param _localStorageService - LocalStorageService
+   * @param _menuService: MenuService
    */
   constructor(
-    private _translateService: TranslateService,
     private _userService: UserService,
-    private _localStorageService: LocalStorageService
+    private _localStorageService: LocalStorageService,
+    private _menuService: MenuService
   ) { }
 
     /** On init hook */
   ngOnInit() {
-    this.initBoard();
+    this.initLinksMenu();
     this._localStorageService.onSet$.subscribe(() => {
-      const link = this.adminMonitoring.entries.find(
+      const link = this.linksMenu.entries.find(
+        (element: any) => element.name === 'Admin & Monitoring'
```

Comment:
Is this value always the same or could be translated?

Reply:
Indeed